cleanup code

zyxkad opened the pull request:
- **cleanup mixins**
- **clean up MagnetBootItem**
- **clean up a lots of code**


and yes github workflow must working this time

blockninja124 commented:
I was going to merge this PR since it seemed good and fixed Powerful thrusters, but in testing I found that magnet boots no longer work? Has something changed and I'm just not doing it right?

zyxkad commented:
> I was going to merge this PR since it seemed good and fixed Powerful thrusters, but in testing I found that magnet boots no longer work? Has something changed and I'm just not doing it right?

Ah so I added a `Enabled` tag, but forgot to set it default to `true`, you may want set that manually by `/data` command
